admin_guide/garbage_collection.adoc

```diff
@@ -86,18 +86,18 @@ Each spin of the garbage collector loop goes through the following steps:
 1. Retrieve a list of available containers.
 2. Filter out all containers that are running or are not alive longer than
 the `*minimum-container-ttl-duration*` parameter.
-3. Determine pod membership of each remaining container.
-4. Remove all unidentified containers (with no pod membership).
-5. For each pod that contains more containers than the
-`*maximum-dead-containers-per-container*` parameter, sort the containers by
+3. Decompose all remaining containers into equivalence classes based on pod and image name membership.
```

Comment:
Decompose is standard mathematical term. Classify is more human and sounds better. Will update it.